content/applications/marketing/events/revenues_report.rst

```diff
@@ -0,0 +1,161 @@
+data and analytics. These reports can either be focused on the *Attendees* or the *Revenues*. Both
+reporting pages offer their own unique array of metrics and options.
+
+The following documentation focuses solely on the reporting options related to event *Revenues*.
+
+Revenues reporting page
+=======================
+
+To access the *Attendees* reporting page in the Odoo *Events* application, navigate to
+:menuselection:`Events app --> Reporting --> Revenues`.
+
+.. image:: revenues_report/default-view.png
+   :align: center
+   :alt: The default view of the Revenues reporting page in the Odoo Events application.
+
+By default, the :guilabel:`Revenues` reporting page appears as a graph (a :icon:`fa-line-chart`
+:guilabel:`(line chart)` with :icon:`fa-database` :guilabel:`(stacked)` data). The default filters,
+:guilabel:`Non-free tickets` and :guilabel:`Event Start Date: (current year)`, are present in the
+:guilabel:`Search...` bar.
+
+.. tip::
+   To learn more about the various graph views (and graph view options), refer to the documentation
+   on :ref:`Graph views <reporting/using-graph>`.
+
+The :guilabel:`Attendees` reporting page can also be viewed as a :ref:`pivot table
+<reporting/views/pivot>`, by clicking the :icon:`oi-view-pivot` :guilabel:`(pivot table)` icon in
+the upper-right corner.
+
+Measures
+--------
+
+Choosing specific :ref:`Measures <reporting/choosing-measures>` is a quick way to customize
+reporting pages.
+
+Regardless of view (graph or pivot table), the measures on the :guilabel:`Revenues` reporting page
+are as follows: :guilabel:`Revenues`, :guilabel:`Untaxed Revenues`, and :guilabel:`Count`.
+
+.. note::
+   In the default graph view of the :guilabel:`Revenues` reporting page, only the
+   :guilabel:`Revenues` option is set in the :guilabel:`Measures` drop-down menu.
+
+   When the pivot option is selected, via the :icon:`oi-view-pivot` :guilabel:`(pivot table)` icon
+   in the upper-right corner, all :guilabel:`Measures` options are selected, by default.
+
+.. tip::
+   In graph view, only one of the :guilabel:`Measures` can be selected at a time.
+
+- :guilabel:`Revenues`: shows the revenues generated from events.
+- :guilabel:`Untaxed Revenues`: shows the untaxed revenues generated from events.
+- :guilabel:`Count`: shows the total amount of registrants who attended events.
+
+Filters and grouping options
+----------------------------
+
+To reveal a drop-down menu of filter and grouping options that can be utilized to create custom
+revenue reports, click the :icon:`fa-caret-down` :guilabel:`(down arrow)` to the right of the
+:guilabel:`Search...` bar.
+
+Doing so opens a drop-down mega menu of options organized into columns: :ref:`Filters
+<search/preconfigured-filters>`, :ref:`Group By <search/group>`, and :ref:`Favorites
+<search/favorites>`.
+
+.. note::
+   If time-related option has been selected from the :guilabel:`Filters` column (e.g. the default
```

Comment:
I don't know if you want to include this, but only one selection can be made from the comparison column at a time. (eg- you can compare start dates from this year vs last year, but you can't compare start dates and registration dates)